circleguard/gui.py

```diff
@@ -896,11 +896,13 @@ def __init__(self):
         self.darkmode = OptionWidget("Dark mode", "Come join the dark side", "dark_theme")
         self.darkmode.box.stateChanged.connect(self.reload_theme)
         self.visualizer_info = OptionWidget("Show Visualizer info", "", "visualizer_info")
-        self.visualizer_bg = OptionWidget("Black Visualizer bg", "Reopen Visualizer for it to apply", "visualizer_bg")
+        self.visualizer_frametime = OptionWidget("Show frametime graph in visualizer", "", "visualizer_frametime")
+        self.visualizer_bg = OptionWidget("Black Visualizer bg", "", "visualizer_bg")
         self.visualizer_bg.box.stateChanged.connect(self.reload_theme)
+        self.visualizer_bg = OptionWidget("Render Hitobjects", "Reopen Visualizer for it to apply", "render_beatmap")
```

Comment:
forgot to add `visualizer_bg` to the settings layout?

Reply:
also hang on this is just using the wrong setting (`render_beatmap`). Something very wrong happened here

Reply:
simple copy and paste error, mb